src/components/Subjects/Datalist.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { PRIMARY_SUBJECTS } from '../../constants/SubjectTypes'
+
+const Datalist = ({ subjectsType }) => {
+  if (subjectsType === PRIMARY_SUBJECTS) {
+    return (
+      <datalist id={subjectsType}>
+        <option value='Biologia' />
```

Comment:
Probably I will get rid of datalist, as it's not compatible with Safari on macOS/Windows and iOS